cpplint.py

```diff
@@ -101,7 +101,9 @@
   To suppress false-positive errors of certain categories, add a
   'NOLINT(category[, category...])' comment to the line.  NOLINT or NOLINT(*)
   suppresses errors of all categories on that line. To suppress categories
-  on the next line use NOLINTNEXTLINE instead of NOLINT.
+  on the next line use NOLINTNEXTLINE instead of NOLINT. To suppress errors in
+  a block of code 'NOLINTBEGIN(category[, category...])' comment to a line at
+  the start of the block and to end the block add a comment with 'NOLINTEND'
```

Comment:
It might be nitpicky, but this should clarify whether the lines of begin and end are also included. So something like`... use 'NOLINTBEGIN(...) comment on the first line of the block, ... use a comment with NOLINT END on the last line of the block

Reply:
The way it is written the BEGIN would include the line it was written on but the END would not include that line in the suppression. It might be cleaner and easier to read to suggest/require the BEGIN and END are on lines by themselves otherwise the current way it is written of inclusive and non-inclusive bounds could be a bit strange. This could possibly be made cleaner by the merging of block and single line error suppression